src/Tribe/Tickets_View.php

```diff
@@ -1121,6 +1121,13 @@ public function get_rsvp_block( $post, $echo = true ) {
 			return $template->template( 'v2/rsvp-kitchen-sink', $args, $echo );
 		}
 
+		/**
+		 * Allow for the addition of content (namely the "Who's Attending?" list) above the ticket form.
+		 *
+		 * @since 4.5.5
+		 */
+		do_action( 'tribe_tickets_before_front_end_ticket_form' );
```

Comment:
Why not use the `Tribe__Template` actions to include?

Reply:
This is not introducing anything new there, we're just making it compatible like it was before. It was late in the sprint to change things beyond repairing the bug itself.